live-examples/css-examples/fragmentation/break-after.html

```diff
@@ -0,0 +1,36 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="break-after">
+    <div class="example-choice">
+        <pre><code class="language-css">break-after: auto;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">break-after: page;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">break-after: left;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
```

Comment:
Same comment as in 'break-before' to highlight the difference in the print preview between `page` and `left`.